app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
+  has_many :posts, foreign_key: 'author_id'
+
+  def recent_3_posts
+    Post.last(3)
+  end
```

Comment:
- [ ] Your implementation here returns the last three (3) posts in ascending order, whereas you are required to return the posts in descending order. For example, if I have 10 posts, your method here returns posts 8, 9 and 10 in that order whereas it should return posts 10, 9 and 8. KIndly fix that.

  **SUGGESTION:** You can change the method to something like so:
  ```
  def recent_3_posts
    Post.order('created_at Desc').limit(3)
  end
  ```